.github/services/webdav/nginx/action.yml

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: nginx
+description: "Behavior test for webdav in nginx"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Setup webdav in nginx
+      shell: bash
+      working-directory: fixtures/webdav
+      run: docker compose -f docker-compose-webdav.yml up -d
+    - name: Setup
+      shell: bash
+      run: |
+        cat << EOF >> $GITHUB_ENV
+        OPENDAL_WEBDAV_ENDPOINT=tcp://127.0.0.1:8080
```

Comment:
https://github.com/apache/incubator-opendal/pull/3379/files#diff-e4ffbc031a1c0461139947d312a686b57bfa7f17d0082ef30c1171770e7be613L61

The original endpoint is http scheme, why TCP here?

Reply:
OK, fixed it.